packages/backend/src/middlewares/sentry.ts

```diff
@@ -11,10 +13,15 @@ export const sentrySetProjectUuidTagMiddleware: RequestHandler = (
     }
 
     if (req.user) {
-        setTags({
-            'user.uuid': req.user.userUuid,
-            'organization.uuid': req.user.organizationUuid,
-        });
+        if (req.user.userUuid && typeof req.user.userUuid === 'string') {
+            setTag('user.uuid', req.user.userUuid);
+        }
+        if (
+            req.user.organizationUuid &&
+            typeof req.user.organizationUuid === 'string'
+        ) {
+            setTag('organization.uuid', req.user.organizationUuid);
+        }
```

Comment:
do we need a runtime check? TS knows those fields exist, I don't understand why eslint fails 🤔

Reply:
because of the eslintrc.js where we reference paths to ignore these rules
```
"@typescript-eslint/no-unsafe-member-access": "off",
                "@typescript-eslint/no-unsafe-assignment": "off",
                "@typescript-eslint/no-unsafe-call": "off"
```
see line 50 - instead of turning off for this directory, I fixed with the checks

Reply:
what I was referring to is that:

typescript gets the type but eslint doesn't
![CleanShot 2025-03-11 at 10 15 01@2x](https://github.com/user-attachments/assets/532214e7-d29a-433f-a457-7178c80f37b3)

eslint doesn't respect this declaration for some reason:
https://github.com/lightdash/lightdash/blob/79f2f78c4793adea82f2da4807deac47bc8a8005/packages/backend/src/App.ts#L70-L88